lib/live_view_demo_web/live/clock_live.ex

```diff
@@ -20,6 +22,11 @@ defmodule LiveViewDemoWeb.ClockLive do
     {:noreply, put_date(socket)}
   end
 
+  def handle_event("boom", "", socket) do
```

Comment:
Clicking the clock sends this event, but there was no handler for it.

Reply:
I remember seeing this in the original demo where it shows how the front-end gracefully recovers after a crash
https://github.com/chrismccord/phoenix_live_view_example/blob/0b8e8ca080a142c5befa0f9c3c0e49956892df32/lib/demo_web/live/counter_live.ex#L8

Reply:
@kevinkoltz oooohhhh thanks!  That make more sense now.